RevenueCatUI/CustomerCenter/Views/RelevantPurchasesListView.swift

```diff
@@ -113,71 +122,104 @@ struct ActiveSubscriptionsListView: View {
 
     @ViewBuilder
     var content: some View {
-        if viewModel.activePurchases.isEmpty {
-            List {
-                let fallbackDescription = localization[.tryCheckRestore]
-
-                Section {
+        ScrollViewWithOSBackground {
+            LazyVStack(spacing: 0) {
+                if viewModel.isEmpty {
                     CompatibilityContentUnavailableView(
                         self.viewModel.screen.title,
                         systemImage: "exclamationmark.triangle.fill",
-                        description: Text(self.viewModel.screen.subtitle ?? fallbackDescription)
+                        description: Text(self.viewModel.screen.subtitle ?? localization[.tryCheckRestore])
+                    )
+                    .padding(.horizontal)
+                    .padding(.vertical)
```

Comment:
I think this is the same as
```suggestion
                    .padding()
```
unless you need different values for horizontal vs vertical